projects/js-packages/publicize-components/src/components/admin-page/index.tsx

```diff
@@ -59,9 +63,10 @@ export const SocialAdminPage = () => {
 	}
 
 	return (
-		<AdminPage moduleName={ moduleName } header={ <AdminPageHeader /> }>
+		<AdminPage moduleName={ moduleName } header={ <AdminPageHeader /> } showFooter={ ! is_wpcom }>
 			<GlobalNotices />
-			{ ( ! hasSocialPaidFeatures() && showPricingPage ) || forceDisplayPricingPage ? (
+			{ ( ! is_wpcom && ! is_woa && ! hasSocialPaidFeatures() && showPricingPage ) ||
```

Comment:
I was thinking we probably shouldn't show the pricing page on WoA either. I think we can see the plan there, but it would make for a more consistent experience.